interfaces/seccomp/backend.go

```diff
@@ -113,11 +113,14 @@ func (b *Backend) combineSnippets(snapInfo *snap.Info, opts interfaces.Confineme
 
 func addContent(securityTag string, opts interfaces.ConfinementOptions, snippets map[string][][]byte, content map[string]*osutil.FileState) {
 	var buffer bytes.Buffer
+	if opts.Classic && !opts.JailMode {
+		// Classic confinement doesn't generate a seccomp profile at all
+		return
+	}
```

Comment:
Would it be better to instead generate one with @unrestricted? Otherwise snap-confine needs to ignore rather than die() when the seccomp profile doesn't exist and I'd much prefer we die() to not accidentally keep going when we shouldn't (in case of coding errors).